src/pages/Profile.tsx

```diff
@@ -98,7 +138,8 @@ const Profile = () => {
       });
     };
     fetchProfileContents();
-  }, [value, user]);
+    window.scrollTo(0, 0);
```

Comment:
Does this scroll to the top? Is it because when you click on a new profile you want it to always be on the top? At the same time if you were just viewing someone else's profile and press the back button you might want to be where you where, so that might complicate things

Reply:
I'd like it so that it resets to the top because if you don't have that scroll to the top on the profile page, it can cause an annoying bug where:

Let's say you have scrolled down on the timeline page, and want to visit a user's profile. Now because you have already scrolled down, the scrollbar keeps its state even on the profile page and it leads to bad UX because it can get confusing unless you are redirected to the top of the profile page.

With all that being considered, I see your point and honestly can't think of a solution for both to coexist just yet

Reply:
Looking back, I think introducing a component that takes care of "scroll to top" at the App level would be better. I assume we want this in most button clicks, UNLESS it is the back button, in which case, we should can incorporate a useEffect to scroll back to a set Scroll position that we need to set beforehand.

What do you think?